software/open-quartermaster.yml

```
@@ -0,0 +1,15 @@
+name: "Open QuarterMaster"
+website_url: "https://openquartermaster.com/"
+source_code_url: "https://github.com/Epic-Breakfast-Productions/OpenQuarterMaster"
+description: "A powerful inventory management system, designed to be flexible and scalable."
+licenses:
+  - GPL-3.0
+platforms:
+  - Deb
+  - Docker
+tags:
+  - Inventory Management
+  - Analytics
+  - Groupware
+  - Manufacturing
+  - Internet of Things (IoT)
```

```suggestion
  - Inventory Management
```

I don't think other tags are relevant